libcpio/include/cpio/cpio.h

```diff
@@ -49,7 +49,12 @@ struct cpio_info {
  * @return             The location of the file in memory; NULL if the index
  *                     exceeds the number of files in the CPIO archive.
  */
-void *cpio_get_entry(void *archive, unsigned long len, int index, const char **name, unsigned long *size);
+void *cpio_get_entry(
```

Comment:
Are you able to open an issue somewhere about proposing a change for adopting this different style and recording it in the style guide? The current guide allows for function arguments to continue on the same line until it reaches the line length I believe.  I believe that we generally try and avoid any stylistic changes that haven't been generated by the style tool to prevent stylistic arguments from sidetracking PRs as everyone has different opinions on what the better style is.  For me, I like more arguments on one line :)

Reply:
I can do this. For a few params, having it on the same line makes sense, especially if the are closely related (e.g. buffer/length). The rationale for having the parameter one on each line it that it allows better diffs/patch/merge workflow, as you (or the tools) can see the actual change better.

Reply:
Agree with @kent-mcleod . Please argue for changing the style on an issue before doing it. We have a style guide already, we shouldn't go an restyle code without buyin, as there are trade-offs that need to be considered. 

Additionally, I believe I turned this *off* in our astyle currently :)